sdk/websites/Azure.ResourceManager.AppService/src/autorest.md

```diff
@@ -136,13 +143,246 @@ rename-rules:
   Ipsec: IPsec|ipsec
   SSO: Sso
   URI: Uri
+  Serverfarm: AppServicePlan
+  serverFarm: AppServicePlan
+  ServerFarm: AppServicePlan
```

Comment:
The `rename-rules` is designed for bulk rename acronym, so not prefer to do this rename here.
Consider to use `rename-mapping` to rename one by one.